.circleci/config.yml

```diff
@@ -295,6 +305,14 @@ workflows:
             - lint_checks
             - unit_tests
             - build_and_push
+      - delete_dependabot_deployment:
```

Comment:
just wondering if not having pointless releases to UAT for dependabot PRs can we not do something like this:

      - deploy_uat:
          context: laa-hmrc-interface-uat
          requires:
            - lint_checks
            - unit_tests
            - build_and_push
          filters:
            branches:
              ignore:
                - /dependabot.*/

Reply:
we still want to make sure the deploy works, we just dont want the pods to hang around once they have been deployed (deploying us part of the test process for new branches)